README.md

```diff
@@ -1 +1,161 @@
+<a name="readme-top"></a>
 
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [Budget App ](#budget-app-)
+  - [video demo 🎥](#video-demo-)
+  - [Render Deploy 🚀](#render-deploy-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Budget App <a name="about-project"></a>
+
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
+
+## video demo 🎥
+
+[video demo here](https://www.loom.com/share/c3197ca55f534b518e79cdd4a7b9b850)
+
```

Comment:
- [ ] Good so far @tanveerisonline but as mentioned before the maximum length of the video should be 5 minutes and no longer than 5 minutes, also you should appear in the video as mentioned in the project requirements 🙃 😉
- [ ] Please kindly record a new video that meets all the requirements 🙏 

<img width="1051" alt="Screenshot 2023-04-29 at 20 28 28" src="https://user-images.githubusercontent.com/74501523/235316109-86c0a7c1-e093-4b78-abca-0b835f2ca9f1.png">